caffe2/observers/time_observer_test.cc

```diff
@@ -67,17 +67,16 @@ unique_ptr<NetBase> CreateNetTestHelper(Workspace* ws) {
 
   return CreateNet(net_def, ws);
 }
-}
+} // namespace
 
 TEST(TimeObserverTest, Test3Seconds) {
   Workspace ws;
   ws.CreateBlob("in");
   NetDef net_def;
   unique_ptr<NetBase> net(CreateNetTestHelper(&ws));
-  unique_ptr<TimeObserver<NetBase>> net_ob =
-      make_unique<TimeObserver<NetBase>>(net.get());
-  const auto* ob = dynamic_cast_if_rtti<const TimeObserver<NetBase>*>(
-      net->AttachObserver(std::move(net_ob)));
+  unique_ptr<TimeObserver> net_ob = make_unique<TimeObserver>(net.get());
```

Comment:
nit: auto for type and caffe2::make_unique for clarity between std:: and caffe2::